packages/react-conformance/src/types.ts

```diff
@@ -2,11 +2,8 @@ import * as React from 'react';
 import { ComponentDoc } from 'react-docgen-typescript';
 import * as ts from 'typescript';
 
-import { defaultTests } from './defaultTests';
```

Comment:
Note that there will be a total rewrite of the react-conformance APIs in the future to try and make them actually good; this change is just making them incrementally less bad.